config/locales/en/lookup.yml

```diff
@@ -10,7 +10,7 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: <h2 class="govuk-heading-s"> Enter a full postcode </h2>
+      input_label: Enter a full postcode 
```

Comment:
This is much better! Can you fix up this commit to the first one?

Reply:
Thanks Leena, i've fixed up the commits now 🙂